config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get '/ingredients', to: 'ingredients#index'
+
+  get '/recipes/:id', to: 'recipes#show'
+  post '/recipes/:id', to: 'recipes#create'
```

Comment:
Similar to the comment I left if your controller. A more RESTful option for this POST is
`post '/recipes/:id/ingredients', to: 'recipe_ingredients#create`